rust/kcl-lib/src/std/revolve.rs

```diff
@@ -169,7 +172,28 @@ async fn inner_revolve(
                     }),
                 )
                 .await?;
+                //TODO: fix me! Need to be able to calculate this to ensure the path isn't colinear
+                glm::DVec2::new(0.0, 1.0)
+            }
+        };
+
+        let mut edge_id = None;
+        // If an edge lies on the axis of revolution it will not exist after the revolve, so
+        // it cannot be used to retrieve data about the solid
+        for path in sketch.paths.clone() {
+            let from = path.get_from();
+            let to = path.get_to();
+
+            let dir = glm::DVec2::new(to[0].n - from[0].n, to[1].n - from[1].n);
+            if glm::are_collinear2d(
+                &dir,
+                &direction,
+                tolerance.as_ref().map(|t| t.to_mm()).unwrap_or(DEFAULT_TOLERANCE),
```

Comment:
Maybe pull this out of the loop? We really only need to do this conversion once for the whole function.

Reply:
Yes